luarules/gadgets/unit_builder_priority.lua

```diff
@@ -614,6 +623,7 @@ local function GetUpdateInterval(teamID)
 	end
 	if maxInterval > 6 then maxInterval = 6 end
 	--Spring.Echo("interval: "..maxInterval)
+	Spring.SetTeamRulesParam(teamID, "builderUpdateInterval", maxInterval)
```

Comment:
This is nice but the bug is still there: maxInterval is being set according to the resource _least_ likely to overflow instead of the resource _most_ likely to overflow.